make-all.sh

```diff
@@ -146,5 +146,16 @@ cd bench/yaxpeax
 $make
 cd ../..
 
-echo "[+] All done!"
+echo "[*] Building sleigh benchmark ..."
+cd bench/sleigh
+cmake -DCMAKE_BUILD_TYPE=Release .
```

Comment:
Any reason for not using a build directory?
```suggestion
cmake -B build -S . -DCMAKE_BUILD_TYPE=Release
```

Reply:
This is following an existing convention. Every other lib (including another that builds with CMake) just builds in the top-level of the lib repo.

I don't object to these changes but if we're going to make them, they should be applied everywhere in a separate change instead of making SLEIGH a special case here.

Reply:
Ah. I should have looked at the whole file. Fair enough then 👍